standalone-container/vespa-jdisc-container.spec

```diff
@@ -0,0 +1,93 @@
+# Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+# Force special prefix for Vespa
+%define _prefix /opt/vespa
+
+# Hack to speed up jar packing for now. This does not affect the rpm size.
+%define __jar_repack %{nil}
+
+Name:           vespa-jdisc-container
+Version:        %version
+BuildArch:      noarch
+Release:        1%{?dist}
+Summary:        Vespa standalone JDisc container
+Group:          Applications/Databases
+License:        Commercial
+URL:            http://vespa.ai
+
+Requires: bash
+Requires: java-1.8.0-openjdk-headless
+
+Conflicts: vespa
+
+%description
+The Vespa standalone JDisc container is a runtime environment for Java
+applications.
+
+%install
+declare jars_dir=%buildroot%_prefix/lib/jars
+mkdir -p "$jars_dir"
+
+declare -a dirs=(
+  jdisc_jetty/target/dependency
+  vespa_jersey2/target/dependency
+)
+for dir in "${dirs[@]}"; do
+  cp "$dir"/* "$jars_dir"
+done
+
+declare -a modules=(
```

Comment:
It seems that you have missed all the jersey2 bundles listed in container-disc/pom.xml.

Reply:
That's actually taken care of 6 lines above: I'm taking all vespa_jersey2 dependencies (which gets me a few more than I actually need).

Reply:
ok, good!